WORKSPACE

```diff
@@ -64,7 +64,7 @@ bind(
 # Used by prometheus-cpp.
 local_repository(
     name = "net_zlib_zlib",
-    path = "tools/zlib"
+    path = "tools/zlib",
```

Comment:
this change isn't really necessary

Reply:
This is true in terms of Python syntax, but buildifier makes this change, and as a result it's necessary in order to pass CI. :)